src/app/shared/object-collection/shared/listable-object/listable-object-component-loader.component.ts

```diff
@@ -51,6 +51,8 @@ export class ListableObjectComponentLoaderComponent implements OnInit {
    */
   @Input() showLabel = true;
 
+  @Input() value: string;
```

Comment:
could you please add a typescript comment?

Reply:
Added typedocs